tests/core/test_test.py

```diff
@@ -40,3 +40,57 @@ async def test_testing_valid_with_non_e2e_core_model(core_agent: Agent):
         "data/test_yaml_stories/test_stories_entity_annotations.yml", core_agent
     )
     assert "report" in result.keys()
+
+
+async def test_action_unlikely_intent_warnings_1(
+    tmpdir: Path, intent_ted_policy_moodbot_agent: Agent
+):
+    result = await rasa.core.test.test(
+        "data/test_yaml_stories/test_action_unlikely_intent_1.yml",
```

Comment:
I wonder if it would be more readable if the stories were inline in this file?

Reply:
that would mean mocking `_create_data_generator` and `TrainingDataImporter.load_from_dict`, and that's something I would like to avoid in order to make these tests simpler

Reply:
you could also just write it to a temporary yaml file.